src/functions.js

```diff
@@ -0,0 +1,101 @@
+import UpdateStorage from './Modules/updateStorage.js';
+import addTask from './Modules/add.js';
+
+const getTodos = () => JSON.parse(localStorage.getItem('storedStTask')) || [];
+
+const displayTasks = (taskInject) => {
+  const tasks = getTodos();
+  taskInject.innerHTML = '';
+  tasks.forEach((task) => {
+    const li = document.createElement('li');
+    li.classList.add('li');
+    li.setAttribute('id', task.index);
+    taskInject.appendChild(li);
+    //
```

Comment:
- [ ]  try as much as possible to avoid leaving unused codes as comments or empty commets in your codebase since it makes your code bulky and slow. check also lines 18, 24, 29, 32, 35 and 38.